src/services/user-config-service.ts

```diff
@@ -39,17 +34,17 @@ export const updateRepoConfig = async (
 
 			if (!repoSyncState) {
 				logger.error({
-					githubInstallationId,
+					githubInstallationId: gitHubInstallationClient.githubInstallationId,
```

Comment:
Nit: seeing we're modifying this file, can we change githubInstallationId > gitHubInstallationId?